### ISSUE TITLE
Questions about hyperparameters

### ISSUE THREAD

**Issue by emjay73** [guoyww/AnimateDiff#127]
Thank you for releasing the code! 
I have some questions regarding the hyperparameters. 
Do the released config files contain the actual hyperparameter values used for the results in the paper? 
I wonder what the actual max_train_epoch value is for image_finetune.yaml and which dataset it was trained on. 
For instance, was the 2M validation set from Webvid used for training instead of the 10M train set? 
Thank you in advance.
